smarts/core/sumo_road_network.py

```diff
@@ -595,8 +600,13 @@ def edges_at_point(self, point: Point) -> Tuple[Point, Point]:
             return left_edge, right_edge
 
         @lru_cache(maxsize=4)
-        def shape(self, width: float = 0.0, buffer_width: float = 0.0) -> Polygon:
-            new_width = width + buffer_width
+        def shape(
+            self, buffer_width: float = 0.0, default_width: Optional[float] = None
+        ) -> Polygon:
+            if default_width is None:
+                new_width = buffer_width
+            else:
+                new_width = default_width + buffer_width
```

Comment:
I have a slight preference for (both here and elsewhere):
```python
new_width = buffer_width
if default_width:
    new_width += default_width
```

(This is a non-blocking _suggestion_ only.)